styles.css

```diff
@@ -1,21 +1,23 @@
 body {
   margin: 0;
   height: 100%;
-  width: 100vw;
-  overflow: hidden;
+  width: 100%;
+  overflow-x: hidden;
+  overflow-y: scroll;
 }
 
 .page-top {
   background-color: #3c3a39;
-  padding: 0 0 0 0;
+
+  /* padding: 0 0 0 0; */
   align-items: center;
   letter-spacing: 0.35px;
 }
 
 .top-nav {
   display: flex;
```

Comment:
Please do not repeat this line of code and use the DRY rule. You can make a class and use it as many times as it is needed.